src/Entities/Pawn.cpp

```diff
@@ -70,7 +70,7 @@ void cPawn::Tick(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
 
 			Vector3d v3Delta = a_Entity->GetPosition() - m_Pusher->GetPosition();
 			v3Delta.y = 0.0;  // we only push sideways
-			v3Delta *= 1.0 / v3Delta.Length();  // we push harder if we're close
+			v3Delta *= 1.0 / (v3Delta.Length() + 0.01);  // we push harder if we're close
```

Comment:
Oh yes... we are talking about vectors so the length is (like Pythagoras said) length l^2 = a^2 + b^2 + c^3 where a,b,c are the coordinates..